toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1183,7 +1206,8 @@ func updateUserPassword(installRoot, username, password string) (err error) {
 	return
 }
 
-func tdnfInstall(packageName, installRoot string, currentPackagesInstalled, totalPackages int) (packagesInstalled int, err error) {
+// TdnfInstall installs a packge in the current environment
+func TdnfInstall(packageName, installRoot string, currentPackagesInstalled, totalPackages int) (packagesInstalled int, err error) {
```

Comment:
>TdnfInstall [](start = 5, length = 11)

Please move this above private function now. I think we didn't follow that everywhere but it's pretty common and makes it easier to find all of the public functions faster.